internal/backend/runtime/omni/controllers/omni/machine_extensions.go

```diff
@@ -83,14 +84,25 @@ func (ctrl *MachineExtensionsController) MapInput(ctx context.Context, _ *zap.Lo
 
 	switch ptr.Type() {
 	case omni.MachineExtensionsType:
-		root, ok := res.Metadata().Labels().Get(labelRootConfiguration)
+		clusterName, ok := res.Metadata().Labels().Get(omni.LabelCluster)
```

Comment:
The bug I discovered:
- `ExtensionsConfiguration` is defined for a machine and generates one `MachineExtensions` for it.
- another `ExtensionsConfiguration` is defined on the cluster level.
- `ExtensionsConfiguration` for a machine is dropped, and removes it's `MachineExtensions` resource.
- Nothing triggers reconcile for `ExtensionsConfiguration` on the cluster level.

Make `MachineExtensions` destroy trigger reconcile for the whole cluster so that other `ExtensionsConfigurations` could regenerate their own `MachineExtensions`.